### PR TITLE
Leadership update

### PR DESCRIPTION
**Pull request by jordangov**
This PR simply swaps out Hunter for Noah on the "staff" section. There are larger plans for a redesign, including getting more team members on the site, but this is important in the interim.

Can I get a quick 👍 @jgarber623-gov ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .jekyll-cache/
 .sass-cache/
 public/
+.DS_Store
```
Comment (jgarber623-gov) [inline .gitignore]: This should be in your global `~/.gitignore`. :grimacing: 

See https://github.com/github/gitignore/blob/master/Global/macOS.gitignore for an example (assuming you're using macOS, of course).

File: src/_data/staff.yml [hunk 2]
```diff
@@ -14,13 +14,13 @@
...
-  img_credit: "Hunter Price / &copy;"
+- name: Noah Firth
+  img_url: assets/img/staff/noah_firth.jpg
+  img_credit: "Noah Firth / &copy;"
   role: Director, Air Force Digital Service
-  previously: Forsa Consulting
+  previously: U.S. Air Force
   quote: |
-    To improve is to change, so to be perfect is to change often. (Sir Winston Churchill)
+    Nothing makes you feel more like a bridge troll than looking through your phone for pictures of yourself. (Noah Firth)
```
Comment (jgarber623-gov) [inline src/_data/staff.yml]: `[ APPROVED ]`

File: src/assets/js/script.js [hunk 3]
```diff
@@ -58,5 +58,14 @@ $(function() {
...
   $mobileNavButton.on('click', toggleMobileNav);
+  
+  $('.staff-members img[title*="Noah"]').hover(
+    function() {
+      $(this).attr('src', 'assets/img/staff/noah_firth_af.jpg');
+    },
+    function() {
+      $(this).attr('src', 'assets/img/staff/noah_firth.jpg');
+    }
+  );
```
Comment (jgarber623-gov) [inline src/assets/js/script.js]: :man_facepalming:

File: .gitignore (hunk 1 above)
Comment (jordangov) [reply .gitignore]: While I agree (and I'll do that), I think adding it to the project gitignore doesn't hurt...?

File: src/_data/staff.yml (hunk 2 above)
Comment (jordangov) [reply src/_data/staff.yml]: Direct quote.

File: src/assets/js/script.js (hunk 3 above)
Comment (jordangov) [reply src/assets/js/script.js]: :D

**Comment by jordangov** [comment]
@jgarber623-gov are we good? We have that same weird Travis CI issue...

**Pull request merged**